### ISSUE TITLE
Synonym correction: cytosolic tRNA wobble base thiouridylase complex (GO:0002144)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-ontology#25725]
The "Cut1-Cut2 complex" synonym here should be "Ctu1-Ctu2 complex".
(e.g. see abstract of PMID: 18391219)

id: GO:0002144
name: cytosolic tRNA wobble base thiouridylase complex
namespace: cellular_component
def: "A complex of two proteins involved in the thiolation of uridine 34 (U34) of tRNAs decoding two-family box triplets." [PMID:17062623, PMID:18391219]
synonym: "Cut1-Cut2 complex" NARROW []
synonym: "tRNA thiouridylase" BROAD []
is_a: GO:0140535 ! intracellular protein-containing complex
is_a: GO:1902494 ! catalytic complex
relationship: part_of GO:0005829 ! cytosol
created_by: hjd
creation_date: 2009-05-08T02:32:55Z

**Comment by pgaudet** [comment]
Right. I think cut1-cut2 is a different complex, see https://pubmed.ncbi.nlm.nih.gov/8978688/

@ValWood can you please confirm? 

Thanks, Pascale

**Comment by ValWood** [comment]
@sjm41 is correct this is ctu1 ctu2, not cut1-cut2 
(cut1 cut2 is separase securin. This is a typo)
